### PR TITLE
Deemix: Add Role

### PR DESCRIPTION
**Pull request by RyanRadly**
Added Deemix role from bocci/deemix Docker image.

### CODE REVIEW

**Comment by chazlarson** [comment]
please rebase against develop

**Comment by RyanRadly** [comment]
Changed ports/image source/variables since docker image changed, rebased against develop.

File: roles/deemix/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    env:
+      BACKUP: "yes"
+      VIRTUAL_HOST: "deemix.{{ user.domain }}"
+      VIRTUAL_PORT: "9666"
+      PUID: "{{ uid }}"
+      PGID: "{{ gid }}"
+      LETSENCRYPT_HOST: "deemix.{{ user.domain }}"
+      LETSENCRYPT_EMAIL: "{{ user.email }}"
+      TZ: "{{ tz }}"
+      ARL: "{{ demix.arl }}"
```
Comment (Migz93) [inline roles/deemix/tasks/main.yml:56]: Change this to deemix

**Comment by Migz93** [comment]
This can be merged if line 56 is changed to "deemix" instead of "demix", it fails as it currently is, the failed appveyor test shows it along with #250 mentioning it. Please test your own branches in future to ensure they work fully before PRing them.

**Comment by Migz93** [comment]
Actually using it further the other issue in #250 is present. It just constantly spams "Settings loaded! and "Logged In" over and over. Can't search for anything, can't download anything.

Please look into that and test that it's working as expected, if it transpires its an issue with the docker itself then we'll need a different image or this one to be fixed before it can be merged.

**Comment by saltydk** [comment]
Closed due to inactivity.

**Pull request closed without merging**